src/components/MAPI/clusters/ClusterDetail/ClusterDetailWidgetProvider.tsx

```diff
@@ -118,22 +125,30 @@ const ClusterDetailWidgetProvider: React.FC<IClusterDetailWidgetProviderProps> =
       >
         {(value) => <Text>{value}</Text>}
       </ClusterDetailWidgetOptionalValue>
-      <ClusterDetailWidgetOptionalValue value={accountID} loaderWidth={300}>
-        {(value) => (
-          <StyledLink
-            color='text-weak'
-            href={accountIDPath}
-            rel='noopener noreferrer'
-            target='_blank'
-          >
-            <code>{value}</code>
-            <i
-              className='fa fa-open-in-new'
-              aria-hidden={true}
-              role='presentation'
-            />
-          </StyledLink>
-        )}
+      <ClusterDetailWidgetOptionalValue
+        value={accountID}
+        loaderWidth={300}
+        replaceEmptyValue={false}
+      >
+        {(value) =>
+          value === '' ? (
+            <NotAvailable />
```

Comment:
Passing in the `NotAvailable` component directly to keep it visually aligned with the "Subscription ID" label.